abdulfatai360/banka
# #165600844 Implement feedback

#### What does this PR do?
Implement feedback

#### Description of Task to be completed?
- Move the `/test` directory inside the `src/` directory
- Give full descriptive names to my functions and methods
- Make my error messages concise and descriptive
- Remove all `console.logs()` in my codebase
- Modify my error handling logic so that they are not tightly coupled to `PostgreSQL`
- Make use of JavaScript ES6 `Class` syntax as much as possible

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/53/head:ch-feedback-implementation-165600844`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-feedback-implementation-165600844`
- Your directory will now be an exact copy of the PR
- Run `npm install` to install all project's dependencies onto your local machine
- Type `npm run build` to run the build script and populate the `dist/` directory containing transpiled code
- Run `npm start` command to start the development server

#### What are the relevant pivotal tracker stories?
[#165600844](https://www.pivotaltracker.com/story/show/165600844)
